tests/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    assert sum(split_integer(15, 3)) == 15
 
 
 def test_should_split_a_number_into_equal_parts_when_value_is_divisible_by_number_of_parts():
-    pass
+    assert split_integer(10, 2) == [5, 5]
 
 
 def test_should_return_a_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    assert split_integer(6, 1) == [6]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal():
-    pass
+    assert split_integer(10, 3) == [2, 4, 4]
```

Comment:
This test is failed with correct reference because it doesn't  satisfy the following condition:
> - The difference between the max and min number in the array should be <= 1